iOS Client/PodcastsController/PodcastDetailViewController.swift

```diff
@@ -79,17 +117,27 @@ class PodcastDetailViewController: UIViewController, StoreSubscriber {
     
     func newState(state: MBAppState) {
         switch state.podcastsState.player {
-        case .error, .initialized, .paused, .finished:
-            playPauseButton.setImage(#imageLiteral(resourceName: "play"), for: .normal)
+        case .initialized, .error, .paused, .finished:
+            playPauseButton.setImage(#imageLiteral(resourceName: "play-arrow"), for: .normal)
+            if playerState == .playing, playerState != .initialized { animateImage(size: CGSize(width: 150, height: 150), duration: 0.6)}
```

Comment:
if `playerState == .playing` then it follows that `playerState != .initialized` b/c `.playing != .initialized`. Thus, we can delete the second check.